webapps/host-manager/css/manager.css

```diff
@@ -125,6 +132,16 @@ TD {
     vertical-align: middle;
     font-family   : sans-serif, Tahoma, Arial;
     color         : black;
+    padding       : 5px;
+    color         : #636363;
```

Comment:
Switching from Black to Grey reduces the contrast and makes it harder to read. I think it would be better left as black. It is also odd to have the titles stand out less than the content.